Update 26.1 (#272)
DevExpress/dotnet-eud: eud-for-devexpress-reports/reporting-for-web/report-designer/preview-print-and-export-reports.md

Reviewer: ```suggestion
To switch a report to print preview mode, click the **Preview** button on the [toolbar](report-designer-tools/toolbar.md). You will see your report populated with data and broken down into pages.
```

Reviewer: ```suggestion
> To learn more about  options available in print preview mode, refer to the [Document Viewer](../document-viewer.md) section of this documentation.
```

Reviewer: ```suggestion
When in Preview mode, you can use toolbar commands to print out your report.
```

Reviewer: ```suggestion
When in Preview mode, you can export your report to files in different formats.
```

Reviewer: ```suggestion
You can specify the **Can Publish Options** setting in the Properties grid to exclude report controls from export formats of your choosing.
```

Reviewer: Which table? Specify.

Author: ```suggestion
The table below describes the following: 
```

Reviewer: ```suggestion
- Roles you can assign to ensure that screen readers correctly identify element purpose in the exported PDF document.
```

![Accessible Role property grid](../images/web-acc-role-property-grid.png)

> Set the **PDF/UA Compatibility** property to **PDF/UA-1** or **PDF/UA-2** to conform the exported PDF document to the PDF/UA specification. Then, export the report to PDF format.

Reviewer: ```suggestion
Set the **PDF/UA Compatibility** property to **PDF/UA-1** or **PDF/UA-2** to conform the exported PDF document to PDF/UA specification. Then, export the report as a PDF.
```

| `Watermark` (an image watermark) | Treated as an artifact; excluded from the PDF logical structure. | Figure | 
| `Watermark` (a text watermark) | Treated as an artifact; excluded from the PDF logical structure. | Paragraph |
| `Picture Box`, `Shape`, `Bar Code`, `Zip Code` | Treated as a `Figure`. | Decorative (Artifact) | 

> | `Page Info` | Treated as an artifact; excluded from the PDF logical structure. | Paragraph (use for meaningful content such as dates or page numbers) |

Reviewer 2: Should we add sections for Page info and Panel as well as for the rest controls below?

- **Document Signature**

    If the control displays a document signature, use the **Accessible Description** option in the control's **PDF Signature Options** in the **Export Options** tab:

    ![Document signature Accessible Description property](../images/web-document-digital-signature-description.png)

- **Signature Placeholder**

    If the control is a signature placeholder (the document signature is disabled), use the control's **Accessible Description** property:

    ![Signature placeholder Accessible Description property](../images/web-xpdfsignature-accessible-description.png)

The following image shows both signatures in the PDF tag tree:

![Digital signatures in the PDF tag tree](../images/digital-signature-in-a-tag-tree.png)

### Limitations

* The **Never Embedded Fonts** export option is not supported for PDF/UA-compatible documents.
* If a report contains an **XRPdfContent** control, you cannot export the report to PDF with the PDF/UA option enabled.
* If a report contains editing fields, the exported document does not comply with PDF/UA requirements.

> * Hyperlinks are exported without the semantic tags required by **PDF/UA-2**.

Reviewer: ```suggestion
* Hyperlinks are exported without semantic tags required by **PDF/UA-2**.
```
